src/MiniCover/Utils/OptimizedStackTrace.cs

```diff
@@ -1,82 +0,0 @@
-using System;
-using System.Linq.Expressions;
-using System.Reflection;
-
-namespace MiniCover.Utils
-{
-    public static class OptimizedStackTrace
-    {
-        public static Func<Func<MethodBase, bool>, MethodBase> GetTestMethod { get; }
-
-        static OptimizedStackTrace()
-        {
-            var stackFrameHelperType = typeof(object).Assembly.GetType("System.Diagnostics.StackFrameHelper");
```

Comment:
That's impressive!

Do you have some numbers about the performance gain?

I'm not sure if the performance gain outcomes the complexity + risks of using private methods.

Reply:
It's splitting by 2 the time to execute so if there is  tests method going to be a good benefit

Reply:
Sounds good! Keep it.